### ISSUE TITLE
Object Detection Example gives different graph from the tutorial one 

### ISSUE THREAD

**Issue by AlexandreBriot** [NVIDIA/DIGITS#1296]
Hello, 

I've been trying to get used to networks learning with DIGITS.
Running tutorial on image classification with MNIST dataset gave results similar to the tuto ones.
But following the object detection tutorial was not that clear to me.

I did not get errors while learning the network but I have some difficulties to interpret my results.
Especially the learning curves which are different from the ones presented in your example 

![capture du 2016-11-25 11 56 58](https://cloud.githubusercontent.com/assets/22516726/20623358/85e52af4-b307-11e6-9509-63acfe6b4115.png)

As you can see, after 30 epochs accuracy is still very low. 
Surprisingly, testing images gives quite good results; letting me think that the graph is wrong.
Furthermore, the initial loss is far larger than the one presented in the tuto.
It should be the exact same since it only depends on the data set and the loss function which are identical to yours.

Another difference is that running ./prepare_kitti_data.py, my dataset size is not the same as yours 
train 6373 images vs 5984
val 1108 images vs 1496

![capture du 2016-11-25 12 18 36](https://cloud.githubusercontent.com/assets/22516726/20623744/6a0f54ec-b309-11e6-819e-3f458ac27c73.png)
 

Do you have any idea why my graph is different from the one expected in the tuto ?
Thanks in advance for your answer
Alex

**Comment by gheinrich** [comment]
This has been asked multiple times.

See for example https://github.com/NVIDIA/DIGITS/issues/1266
